frontend/src/metabase-types/types/Metric.ts

```diff
@@ -1,6 +1,7 @@
 import { TableId } from "./Table";
 
-export type MetricId = number;
+type GoogleAnalyticsMetricId = string;
+export type MetricId = number | GoogleAnalyticsMetricId;
```

Comment:
According to an old comment in the `columnName` method of `Metric`, a `string` id is a possibility, so I'm adding it here.